cmd/apps.go

```diff
@@ -50,7 +50,7 @@ func (d DeisCmd) AppCreate(id, buildpack, remote string, noRemote bool) error {
 
 	if !noRemote {
 		if err = git.CreateRemote(s.Client.ControllerURL.Host, remote, app.ID); err != nil {
-			if err.Error() == "exit status 128" {
+			if strings.Contains(err.Error(), "Error when running 'git remote add") {
```

Comment:
This works, but it'd prefer to check for `fatal: remote <remote name> already exists.`, which is more explicit.